subprojects/docs/src/docs/userguide/api/kotlin_dsl.adoc

```diff
@@ -740,55 +740,6 @@ You can silence the warning by setting the `experimentalWarning` property of the
 include::sample[dir="snippets/kotlinDsl/kotlinDslPlugin",files="buildSrc/build.gradle.kts[tags=disable-experimental-warning]"]
 ====
 
-[[kotdsl:precompiled_plugins]]
```

Comment:
❓ I think we should add some http redirection on `docs.gradle.org` to support existing direct links to this after the move.

Hey @eriwen, do you still manage those? if not, who could help?

Reply:
@eskatos I don't think we can fix anchors with a redirect.  We can only redirect to an entirely different page if this page moved.

All of the old docs will still work, so this should only be a problem if we have non-versioned links to this.  If you think this is a serious problem, we can leave the anchor here and just refer to the new chapter/section as a link. WDYT?